src/app.module.ts

```diff
@@ -11,10 +11,10 @@ import { ProductsModule } from './products/products.module';
     TypeOrmModule.forRoot({
       type: 'mysql',
       host: 'localhost',
-      port: 3307,
+      port: 3306,
       username: 'root',
-      password: 'darsh123',
```

Comment:
Please use the same password for our testing database. So we donot need to fix that everytime